packages/primitives/src/primitives/load.js

```diff
@@ -206,10 +196,10 @@ export function load(scopedContext = {}) {
     scopedContext: {
       global: { ...scopedContext },
       ...scopedContext,
-      ...streamsImpl,
       ...urlImpl,
       ...abortControllerImpl,
       ...eventsImpl,
```

Comment:
nit: I noticed `eventsImpl` is still being used in this line despite it's removed in other diffs. is this intentional?

Reply:
That's because `eventsImpl` is just holding `FetchEvent` and `PromiseRejectionEvent` constructor, so if you don't need once of them then you don't need to use them at `eventsImpl` since the rest of constructors (ReadableStream, ReadableStreamBYOBReader, etc) are available in the global context